ceph_iscsi_config/gateway.py

```diff
@@ -26,46 +26,46 @@ def __init__(self, logger, config, name=None):
         else:
             self.hostname = this_host()
 
-    def ceph_rm_blacklist(self, blacklisted_ip):
+    def ceph_rm_blocklist(self, blocklisted_ip):
         """
-        Issue a ceph osd blacklist rm command for a given IP on this host
-        :param blacklisted_ip: IP address (str - dotted quad)
+        Issue a ceph osd blocklist rm command for a given IP on this host
+        :param blocklisted_ip: IP address (str - dotted quad)
         :return: boolean for success of the rm operation
         """
 
-        self.logger.info("Removing blacklisted entry for this host : "
-                         "{}".format(blacklisted_ip))
+        self.logger.info("Removing blocklisted entry for this host : "
+                         "{}".format(blocklisted_ip))
 
         conf = settings.config
         try:
             subprocess.check_output("ceph -n {client_name} --conf {cephconf} "
-                                    "osd blacklist rm {blacklisted_ip}".
-                                    format(blacklisted_ip=blacklisted_ip,
+                                    "osd blocklist rm {blocklisted_ip}".
+                                    format(blocklisted_ip=blocklisted_ip,
                                            client_name=conf.cluster_client_name,
                                            cephconf=conf.cephconf),
                                     stderr=subprocess.STDOUT, shell=True)
         except subprocess.CalledProcessError as err:
-            self.logger.critical("blacklist removal failed: {}. Run"
+            self.logger.critical("blocklist removal failed: {}. Run"
```

Comment:
You will need to retry using the "osd blacklist rm" command if this throws to ensure backwards compatibility

Reply:
done